src/js/actions/guides.js

```diff
@@ -313,6 +313,36 @@ define(function (require, exports) {
     setGuide.transfers = [resetGuidePolicies, deleteGuide];
 
     /**
+     * Clears all the guides in the given document
+     *
+     * @param {{id: number}} document Document model or an object containing document ID
+     * @return {Promise}
+     */
+    var clearGuides = function (document) {
+        if (document === undefined) {
```

Comment:
I had a line in a past PR that way and @iwehrman suggested I use === undefined check instead...

I like the explicitness of this style.